src-docs/src/views/combo_box/combo_box_example.js

```diff
@@ -347,5 +351,27 @@ export const ComboBoxExample = {
       props: { EuiComboBox },
       demo: <Async />,
     },
+    {
+      title: 'Starting With',
```

Comment:
You haven't updated the docs based on the prop name change

```suggestion
      title: 'Sorting matches',
```